scipy/stats/stats.py

```diff
@@ -8121,6 +8121,237 @@ def brunnermunzel(x, y, alternative="two-sided", distribution="t",
     return BrunnerMunzelResult(wbfn, p)
 
 
+def _confint_lowerbound(n, quantile, confidence):
+    r"""
```

Comment:
Please start the docstrings with a single line summary separated from the main documentation with a blank line.

From [numpydoc standards](https://numpydoc.readthedocs.io/en/latest/format.html):
![image](https://user-images.githubusercontent.com/6570539/120056173-6b1bb900-bfef-11eb-94d3-8ef891917df6.png)